explorer/src/entity/JobRun.ts

```diff
@@ -103,9 +103,16 @@ const searchBuilder = (
   params: ISearchParams
 ): SelectQueryBuilder<JobRun> => {
   let query = db.getRepository(JobRun).createQueryBuilder('job_run')
-
   if (params.searchQuery != null) {
-    const searchTokens = params.searchQuery.split(/\s+/)
+    let searchTokens = params.searchQuery.split(/\s+/)
+    searchTokens = searchTokens.map(token => {
+      if (
+        token.substring(0, 2) === '0x' &&
+        (token.length === 66 || token.length === 42)
+      )
+        return token.substring(2)
```

Comment:
Hey there, @NavyAdmiral. Can we place this in a method, and instead of normalizing the searchTokens for all fields, only do it for relevant fields. ie:

```javascript
    query = query
      .where('job_run.runId IN(:...searchTokens)', { searchTokens })
      .orWhere('job_run.jobId IN(:...searchTokens)', { searchTokens })
      .orWhere('job_run.requester IN(:...searchTokens)', { normalizeHashes(searchTokens) })
      .orWhere('job_run.requestId IN(:...searchTokens)', { searchTokens })
      .orWhere('job_run.txHash IN(:...searchTokens)', { normalizeHashes(searchTokens) })
```
I think it's ok that we redundantly do it twice, but feel free to cache it above if you want.